src/makeCalculator.js

```diff
@@ -37,7 +37,46 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    add: function(num) {
+      result += num;
+
+      return calculator;
+    },
+    subtract: function(num) {
+      result -= num;
+
+      return calculator;
+    },
+    multiply: function(num) {
+      result *= num;
+
+      return calculator;
+    },
+    divide: function(num) {
+      result /= num;
+
+      return calculator;
+    },
+    reset: function() {
+      result = 0;
+
+      return calculator;
```

Comment:
- it's better to return `this` instead of `calculator` as it will make a code more scalable 
- only reset and operate should return this, other method should't. Check again the example.

<img width="506" alt="Screenshot 2023-10-24 at 17 34 30" src="https://github.com/mate-academy/js_advanced_calculator/assets/64746943/1c8dc9a4-8f72-4e88-bdbe-60b17ac2501e">

Reply:
still not fixed comments